Split/Network/Security/TlsPinChecker.swift

```diff
@@ -34,30 +34,48 @@ enum CredentialValidationResult: CaseIterable {
 
     var description: String {
         switch self {
-        case .success:
-            return "success"
-        case .error:
-            return "Error validating credentials"
-        case .noPinsForDomain:
-            return "No pins found for domain"
-        case .invalidChain:
-            return "Key chain invalided"
-        case .credentialNotPinned:
-            return "Credential is not pinned"
-        case .spkiError:
-            return "Unable to get SPKI from public key"
-        case .noServerTrustMethod:
-            return "Validation method is not Server Trust"
-        case .unavailableServerTrust:
-            return "No server trust available"
-        case .invalidCredential:
-            return "Invalid credentials"
-        case .invalidParameter:
-            return "Incorrect credentials type"
+            case .success:
+                return "success"
+            case .error:
+                return "Error validating credentials"
+            case .noPinsForDomain:
+                return "No pins found for domain"
+            case .invalidChain:
+                return "Key chain invalided"
+            case .credentialNotPinned:
+                return "Credential is not pinned"
+            case .spkiError:
+                return "Unable to get SPKI from public key"
+            case .noServerTrustMethod:
+                return "Validation method is not Server Trust"
+            case .unavailableServerTrust:
+                return "No server trust available"
+            case .invalidCredential:
+                return "Invalid credentials"
+            case .invalidParameter:
+                return "Incorrect credentials type"
         }
     }
 }
 
+@objc public class CertificatePinningCompleteStatus: NSObject {
+    let host: String
+    let status: CertificatePinningStatus
+    let reason: String
+    
+    public init(host: String, status: CertificatePinningStatus, reason: String) {
+        self.host = host
+        self.status = status
+        self.reason = reason
+    }
+}
+
+@objc public enum CertificatePinningStatus: Int {
+    case success
+    case failed
+    case defaultHandling
+}
```

Comment:
I think it may be good to add some raw values for convenience.
```suggestion
@objc public enum CertificatePinningStatus: String {
    case success = "Success"
    case failed = "Failed"
    case defaultHandling = "Default"
}
```